README.md

```diff
@@ -1 +1,106 @@
-# walk-safe-frontend
+# Walk Safe
+[View Live Application Here](https://walk-safe-frontend.herokuapp.com/)
+
+## Table of Contents
+* [Description](#description)
+* [Learning Goals](#LearningGoals)
+* [Installation](#installation)
+* [Walkthrough](#walkthrough)
+* [Authors](#authors)
+* [Technologies](#technologies)
+* [Demos](#demos)
+
+## Description
+
+Created to empower individuals to walk alone, Walk Safe brings a user's community along from the trip. Utilizing Geolocation and SMS messaging APIs, we keep a user's community informed of their trip status from start to finish. Should an unforeseen trip interruption arise, Walk Safe will immediately alert the user's contact. We've got your back.
+
+## Walkthrough
+
+
+
+## Learning Goals
+
+
+
+*Minimum Viable Product (MVP)*:  
+
+
+## Notable Features
+
+- GoogleMaps Places API to autocomplete start and end point address
+- GoogleMaps API data received from server to determine trip ETA
+- Twilio SMS Messaging API to send start, trip extended, trip end, and alert messaging to user's contact
+- Apollo + GraphQL to receive contact and trip data from server
+- React Countdown timer to provide visual displays for trip ETA data
+- React + React Hooks to navigate user flow
+- AddContact component enabling user to create custom contacts to send SMS Messaging to
+
+
+## Installation
+
+* The application is deployed to Heroku, to view the live application: https://walk-safe-frontend.herokuapp.com/
+
+* To view the application on your local device:
+
+1. Clone down this application
+2. Enter `npm i` in your terminal
+3. Enter `npm start` in your terminal
+4. To view cypress tests open separate tab an enter `npx cypress run`
+
+## Front End Authors
+<table>
+    <tr>
+        <td> Caroline Eubanks <a href="https://github.com/cmeubanks">GH</td>
+    </tr>
+    <tr>
+        <td><img src="https://avatars.githubusercontent.com/u/73092355?v=4" alt="C. Eubanks" width="125" height="auto" /></td>
+    </tr>
+
+    <tr>
+        <td> Bryan Hohn <a href="">GH</td>
+    </tr>
+    <tr>
+        <td><img src="" alt="B. Hohn" width="125" height="auto" /></td>
+    </tr>
+
+    <tr>
+        <td> Peter Muellerleile <a href="">GH</td>
+    </tr>
+    <tr>
+        <td><img src="" alt="P. Muellerleile" width="125" height="auto" /></td>
+    </tr>
+</table>
+
+## Technologies
+<table>
+    <tr>
+        <td>JavaScript</td>
```

Comment:
Nice, love the inclusion of tech logos.  If you're missing any I've got a bunch more - Hooks, TravisCI, etc.  Do we want to include BE tech here as well?